### DISCUSSION TITLE
Transferring ownership of a project to another organization

### DISCUSSION THREAD

**Discussion by iam-rohid** [supabase/supabase#19480] [Feature Requests]
### Discussed in https://github.com/orgs/supabase/discussions/6681

<div type='discussions-op-text'>

<sup>Originally posted by **Olyno** May  3, 2022</sup>
# Feature request

## Is your feature request related to a problem? Please describe.

Currently, it is not possible to transfer your project to other users. This can be very useful in case we test things on a database, and then want to use it in production.

## Describe the solution you'd like

A simple modal where we would just have to specify the email of the new owner, and confirm by email our action (or reconnect). Something similar to Github.

## Describe alternatives you've considered

/

## Additional context

Many developers ask for this feature (e.g from @netgfx, [last August](https://github.com/supabase/supabase/discussions/2814_)), and it is not yet tracked, which is why i created this issue.
</div>

**Comment by Bruh-Codes** [comment]
I want to comment on this feature to make it more relevant. I am finished building a product for my client and i realised they cant own the database. I was disappointed you can only transfer projects to your own managed org. I want my clients to own their own database if they want to upgrade to pro or not that is their business
